modules/claims_api/app/workers/claims_api/claim_establisher.rb

```diff
@@ -18,6 +20,26 @@ def perform(auto_claim_id) # rubocop:disable Metrics/MethodLength
       form_data = auto_claim.to_internal
       auth_headers = auto_claim.auth_headers
 
+      if auth_headers['va_eauth_birlsfilenumber'].blank? || auth_headers['va_eauth_pid'].blank?
+        # call MPI to get missing values
+        mpi = veteran_from_headers(auth_headers)&.mpi
+        auth_headers['va_eauth_pid'] = mpi&.mvi_response&.profile&.participant_id
+        auth_headers['va_eauth_birlsfilenumber'] = mpi&.mvi_response&.profile&.birls_id
+
+        if auth_headers['va_eauth_pid'].blank? || auth_headers['va_eauth_birlsfilenumber'].blank?
```

Comment:
nit: the fact that these `if` statements have a reversed order .... *twitches*

Reply:
IDK how I missed that, fixed lol